src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    return callback(number);
```

Comment:
The `operate` method should update the `result` property by calling the callback with the current `result` and the `number` as arguments. Also, ensure that the callback is a function before invoking it. You can check this using `typeof callback === 'function'`.